README.md

```diff
@@ -1,24 +1,48 @@
-# README
+# Junk4Dollars
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+[![Build Status](https://travis-ci.com/riccjohn/junk4dollars-api.svg?branch=master)](https://travis-ci.com/riccjohn/junk4dollars-api)
 
-Things you may want to cover:
+# Installation
 
-* Ruby version
+## Rails
 
-* System dependencies
+Install `rails 6.0.0`
 
-* Configuration
+```zsh
+gem install rails -v 6.0.0
+```
 
-* Database creation
+Clone this repo
 
-* Database initialization
+```zsh
+git clone git@github.com:riccjohn/junk4dollars-api.git
+```
 
-* How to run the test suite
+Install dependencies
 
-* Services (job queues, cache servers, search engines, etc.)
+```zsh
+cd junk4dollars-api && bundle install
+```
 
-* Deployment instructions
+## Starting the server locally
 
-* ...
+From within the `junk4dollars-api` directory
+
+```zsh
+rails server
+```
+
+Visit `localhost:3000` in your browser
+
+## CI
+
+[Travis CI](https://travis-ci.com/riccjohn/junk4dollars-api)
+
+---
+
+# API
+
+| Route        | Usage                                |
```

Comment:
Nice job documenting the API. If you'd like, we can look into ways of auto-generating this and/or tying it more closely to the code, so you don't have to duplicate it.

Reply:
That'd be great!

Reply:
Awesome. Let's make a Trello card and put it in the backlog so we can keep track of the idea!